hcl/add-default_tags/add-default-tags.spec.yaml

```diff
@@ -0,0 +1,41 @@
+# This batch change uses comby to add default tags to Terraform locals.tf file
+# The comby replace patterns are stored in a TOML file for convenience https://comby.dev/docs/configuration#toml-format
+
+name: add-repo-tags
+description: |
+  Use comby to add the repository name as a ServiceName tag
+
+on:
+  - repositoriesMatchingQuery: file:locals.tf$ lang:HCL tag not ServiceName
+
+steps:
+  - run: |
+      comby -in-place -config  /tmp/patterns.toml;
+      ls
```

Comment:
Is this `ls` command needed?

Reply:
I was really tempted to say "yes it is" and resolve.